lib/go/thrift/json_protocol.go

```diff
@@ -524,6 +524,8 @@ func (p *TJSONProtocol) TypeIdToString(fieldType TType) (string, error) {
 		return "set", nil
 	case LIST:
 		return "lst", nil
+	case UUID:
+		return "uid", nil
```

Comment:
apparently we don't have a written spec for TJSONProtocol, so I followed this part from the implementation of java library.